src/components/MailchimpSignup.js

```diff
@@ -1,65 +1,59 @@
-import emailValidator from 'email-validator'
-import {
-  Anchor,
-  Box,
-  Button,
-  CheckBox,
-  FormField,
-  Paragraph,
-  Text,
-  TextInput,
-} from 'grommet'
-import { Mail } from 'grommet-icons'
 import React, { useEffect, useState } from 'react'
+import emailValidator from 'email-validator'
+import styled from 'styled-components'
+import { Anchor, Box, Button, Paragraph, TextInput } from 'grommet'
+import { MailOption } from 'grommet-icons'
 
-export function MailchimpSignup(props) {
+const StyledButton = styled(Button)`
```

Comment:
please please please, before you start styling everything on your own in pixels, start modifying the grommet theme file so all rhythms stay the same : e.g https://v2.grommet.io/button#button.padding.vertical